src/compiler/crystal/tools/doc/highlighter.cr

```diff
@@ -69,7 +69,7 @@ module Crystal::Doc::Highlighter
                :lib, :fun, :type, :struct, :union, :enum, :macro, :out, :require,
                :case, :when, :select, :then, :of, :abstract, :rescue, :ensure, :is_a?,
                :alias, :pointerof, :sizeof, :instance_sizeof, :offsetof, :as, :as?, :typeof, :for, :in,
-               :undef, :with, :self, :super, :private, :asm, :nil?, :protected, :uninitialized, "new",
+               :with, :self, :super, :private, :asm, :nil?, :protected, :uninitialized, "new",
```

Comment:
Missed the `"new"` on the very same line :grimacing: 
What's up with it, anyway?

Other than that, I also found these omissions from this list:

* `responds_to?`
* arguably `previous_def` (on the same rights as `super`)
* arguably `forall`.

Reply:
Should I make another PR for them?